packages/dataviews/src/dataviews-layouts/grid/use-grid-style.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { useViewportMatch } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import type { Density, ViewGrid } from '../../types';
+
+export default function useGridStyle( view: ViewGrid ) {
+	const isXHuge = useViewportMatch( 'xhuge' );
+	const isHuge = useViewportMatch( 'huge' );
+	const isXlarge = useViewportMatch( 'xlarge' );
+	const isMedium = useViewportMatch( 'small' );
+	// The `balanced` density (default) is handled with css. If another density is selected,
```

Comment:
```suggestion
	// The `medium` density (default) is handled with css. If another density is selected,
```

Reply:
Why `medium`? We've changed to `balanced`.

Reply:
I just saw that we use compact/medium/comfortable in the toggle control. I now realize the types also say `balanced` instead of `medium`. It'd be good to consolidate everything into the same values.

Reply:
Sorry about that. That was a mistake when I reverted the toggleGroup.